pkgs/development/libraries/zchunk/darwin.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/meson.build b/meson.build
```

Comment:
https://github.com/zchunk/zchunk/issues/24 mentions that in theory they have osx. I wonder if we can contribute that.

(would still be required here until they merge)

Reply:
I am seeing both the endian and argp problems mentioned in that issue, but I don't see a fix in any commits. It looks like someone figured it out without making a pull request. So yes, it would make sense for me to make a pull request there too, and then eventually remove the patch here.

Reply:
I don't think that's a blocker, but I think that would be smart.
Otherwise we'll just have to adapt the patch for future updates when it would be easier to just have it done upstream.